client/components/Events/EventScheduleSummary/index.tsx

```diff
@@ -49,11 +59,35 @@ export const EventScheduleSummary = ({schedule, noPadding, forUpdating, useEvent
         currentDateLabel = gettext('Current Date (Based on Event timezone)');
     }
 
+    const start = moment(eventSchedule.start);
+    const end = moment(eventSchedule.end);
```

Comment:
in case there is no end time or for start date when it's all day event it needs to use `moment.utc()` instead of `moment()` to interpret the dates as it is and not trying to convert those to local timezone.
same like here https://github.com/superdesk/newsroom-core/pull/231